"the nautilus script shouldn't be distributed at all these days, no-one is maintaining it"
<https://bitbucket.org/tortoisehg/thg/issue/2178/setuppy-assumes-that-usr-is-writeable>
--- setup.py.orig	2012-09-05 10:36:36.000000000 -0500
+++ setup.py	2013-01-07 17:00:28.000000000 -0600
@@ -152,7 +152,7 @@
         pyqtfolder = os.path.dirname(PyQt4.__file__)
         path.append(os.path.join(pyqtfolder, 'bin'))
         os.putenv('PATH', os.pathsep.join(path))
-        if os.system('pyrcc4 "%s" -o "%s"' % (qrc_file, py_file)) > 0:
+        if os.system('pyrcc4-2.7 "%s" -o "%s"' % (qrc_file, py_file)) > 0:
             self.warn("Unable to generate python module %s for resource file %s"
                       % (py_file, qrc_file))
             if not exists(py_file) or not file(py_file).read():
@@ -421,8 +421,6 @@
     _data_files += [(os.path.join('share', root),
         [os.path.join(root, file_) for file_ in files])
         for root, dirs, files in os.walk('locale')]
-    _data_files += [('/usr/share/nautilus-python/extensions/',
-                     ['contrib/nautilus-thg.py'])]
 
     # Create a config.py.  Distributions will need to supply their own
     cfgfile = os.path.join('tortoisehg', 'util', 'config.py')
